guide/blueprints/config-files.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Uploading Script and Configuration Files
+layout: website-normal
+toc: ../guide_toc.json
+categories: [use, guide, defining-applications]
+---
+
+Blueprints often require that parameterized scripts and configuration files are available to be copied to the
+target VM. These must be URLs resolvable from the Brooklyn instance, or on the Brooklyn classpath. One simple way 
+to achieve this is to compile the support files into a .jar, which is then added to Brooklyn's 'dropins' folder. 
+Alternatively, an OSGi bundle can be used, referenced from the catalog item. 
```

Comment:
Perhaps mention that these methods are specific to the way brooklyn is launched.